NRC.nl.js

```diff
@@ -0,0 +1,157 @@
+{
+	"translatorID": "8e708ceb-ce42-446f-bffd-21e2bdeb0742",
+	"label": "NRC.nl",
+	"creator": "Martijn Staal",
+	"target": "^https?://www.nrc.nl",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-06-15 13:47:40"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	Copyright © 2023 Martijn Staal
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/nieuws/')) {
+		return "newspaperArticle";
+	}
+
+	return false;
+}
+
+function doWeb(doc, url) {
+	return scrape(doc, url)
+}
+
+function scrape(doc, url) {
+	let item = new Zotero.Item("newspaperArticle");
+
+	item.title = doc.querySelector("meta[property='og:title']").content
+	item.language = "nl"
+	item.url = doc.querySelector("meta[property='og:url']").content
+	item.abstractNote = doc.querySelector("meta[property='og:description']").content
+	item.rights = doc.querySelector("meta[name='dcterms.rights']").content
+	item.date = doc.querySelector("meta[property='og:url']").content.match(/\d\d\d\d\/\d\d\/\d\d/)[0].replace(/\//g, "-");
+
```

Comment:
We'd want authors where they're available, e.g. Clara van de Wiel for https://www.nrc.nl/nieuws/2023/06/15/lot-van-natuurherstelwet-timmermans-blijft-erg-onzeker-na-belangrijke-stemming-a4167267